projects/core/src/configurator/commons/facade/configurator-commons.service.ts

```diff
@@ -25,6 +34,13 @@ export class ConfiguratorCommonsService {
     );
   }
 
+  isConfigurationReady(productCode: string): Observable<boolean> {
```

Comment:
Is this method needed, I see no consumer

Reply:
Not found.